LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015
```

Comment:
this has been going so long it is 2021 now 😱

Reply:
Updated!